docs/reference/commandline/run.md

```diff
@@ -139,13 +139,14 @@ file when `docker run` exits.
 
     $ docker run -t -i --rm ubuntu bash
     root@bc338942ef20:/# mount -t tmpfs none /mnt
-    mount: permission denied
+    mount: block device none is write-protected, mounting read-only
+    mount: cannot mount block device none read-only
```

Comment:
@HackToday  I think `mount: permission denied` is correct, because the capability `SYS_ADMIN` is dropped by default which is required to mount.
<pre><code>$ docker run -ti --rm ubuntu bash
root@b2b9573234a4:/# mount -t tmpfs none /mnt
mount: permission denied
root@b2b9573234a4:/#</code></pre>